resources/css/filament/admin/theme.css

```
@@ -200,3 +200,23 @@
     border: none;
     border-radius: 50%;
 }
+
+.fi-page-sub-navigation-sidebar-ctn {
+    @apply bg-white border border-gray-950/5 rounded-xl dark:bg-gray-900 dark:border-white/10;
+}
+
+.fi-page-sub-navigation-sidebar {
+    @apply p-4;
+}
+
+.fi-page-sub-navigation-sidebar > li > ul > li a .fi-sidebar-item-icon {
+    display: none;
+}
+
+.fi-page-sub-navigation-sidebar > li > ul > li a .fi-sidebar-item-grouped-border {
+    display: none;
+}
+
+.fi-page-sub-navigation-sidebar > li > ul > li a {
+    @apply p-3;
+}
```

The selectors are a little brittle, these should be easier to maintain

Also I think each item now consumes too much space so I suggest removing the padding change

```suggestion
.fi-page-sub-navigation-sidebar .fi-sidebar-item-icon {
    @apply hidden;
}

.fi-page-sub-navigation-sidebar .fi-sidebar-item-grouped-border {
    @apply hidden;
}
```

---

Hey @ankit-canyon please look carefully at these changes, I removed the last block and changed the selector on the first two.

---

Yep corrected !